docs/v0.6/en/user-guide/client-libraries/go.md

```diff
@@ -18,16 +18,17 @@ concurrent-safe library that is easy to use with the metric struct.
 Use the following command to install the GreptimeDB client library for Go:
 
 ```shell
-go get github.com/GreptimeTeam/greptimedb-client-go@v0.1.2 google.golang.org/grpc google.golang.org/grpc/credentials/insecure
+go get -u github.com/GreptimeTeam/greptimedb-ingester-go
 ```
 
 Import the library in your code:
 
 ```go
 import (
-    greptime "github.com/GreptimeTeam/greptimedb-client-go"
-    "google.golang.org/grpc"
-    "google.golang.org/grpc/credentials/insecure"
+    "github.com/GreptimeTeam/greptimedb-ingester-go/client"
+    "github.com/GreptimeTeam/greptimedb-ingester-go/config"
+    "github.com/GreptimeTeam/greptimedb-ingester-go/table"
+    "github.com/GreptimeTeam/greptimedb-ingester-go/table/types"
```

Comment:
`client` and `config` has been removed in this [feat: merge client with streamClient](https://github.com/GreptimeTeam/greptimedb-ingester-go/pull/23). But it is ok here, and I will commit a PR to keep it up to date afterwards.